src/pages/workspace/companyCards/WorkspaceCompanyCardsPage.tsx

```diff
@@ -155,7 +155,7 @@ function WorkspaceCompanyCardPage({route}: WorkspaceCompanyCardPageProps) {
             )}
             {!isLoading && (
                 <WorkspacePageWithSections
-                    shouldUseScrollView={isFeedAdded}
+                    shouldUseScrollView
```

Comment:
@narefyev91 Hey, why have you updated it? I'm asking, cause I've [updated](https://github.com/Expensify/App/pull/49928/files#diff-4945562ec6c3a981473a046929178e2a8d2eec354231cc7040a44356c765625dR72) it in my PR as well to prevent warning on ios/android.

Reply:
@VickyStash on small devices - area was not scrollable and button was not visible (no chance to start the flow)

Reply:
@narefyev91 But with this it should work as expected, right?
```
shouldUseScrollView={!isFeedAdded}
```

Reply:
probably yeah - will also add that in my PR